ape_etherscan/utils.py

```diff
@@ -1,4 +1,4 @@
-# TODO: Remove in 0.9 and make this a calculated property.
+# TODO: (deprecated) Remove in 0.9 and make this a calculated property.
```

Comment:
I am wondering if we only need etherscan API if using V2 then

- [ ] figure out